lms/djangoapps/verify_student/views.py

```diff
@@ -946,6 +947,34 @@ def _send_email(user_id, subject, message):
     user.email_user(subject, message, from_address)
 
 
+def _set_user_requirement_status(attempt, namespace, status, reason=None):
+    """Sets the status of a credit requirement for the user,
+    based on a verification checkpoint.
+    """
+    try:
+        checkpoint = VerificationCheckpoint.objects.get(photo_verification=attempt)
+    except VerificationCheckpoint.DoesNotExist:
+        log.warn("Unable to find checkpoint for %s", attempt.user.username)
+        return False
```

Comment:
Since we are just setting status of credit requirement, we don't need a `return` value.